mm2src/mm2_main/src/lp_ordermatch/new_protocol.rs

```diff
@@ -282,6 +285,9 @@ pub struct MakerReserved {
     #[serde(default)]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub rel_protocol_info: Option<Vec<u8>>,
+    #[serde(default)]
+    #[serde(skip_serializing_if = "SwapVersion::is_legacy")]
+    pub swap_version: SwapVersion,
```

Comment:
Same for this. You can check this PR https://github.com/KomodoPlatform/komodo-defi-framework/pull/2353 for a different test template other than the one mentioned in the previous comment.

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2324/commits/8b76118be5e98f3c1bf38f59cf39ec82dfe6b250